profiler/lib.profiler/src/org/netbeans/lib/profiler/heap/CacheDirectory.java

```diff
@@ -28,16 +28,19 @@
  */
 class CacheDirectory {
     
-    private static final String DIR_EXT = ".hwcache";   // NOI18N
+    private static final String DIR_EXT = ".nbcache";   // NOI18N
```

Comment:
This change is useful to avoid clash between Apache NetBeans profiler library and VisualVM profiler library. Both of these libraries are derived from NetBeans 8.2 profiler library, but evolve independently. Sharing the same directory leads to errors like:
```
java.io.IOException: Invalid HPROF version 3 loaded 2
        at ...HprofHeap.<init>(HprofHeap.java)
        at ...HeapFactory.loadHeap(HeapFactory.java)
        at ...HeapFactory.createHeap(HeapFactory.java)
```